pkg/hubble/filters/httpheader.go

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Hubble
+
+package filters
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	flowpb "github.com/cilium/cilium/api/v1/flow"
+	v1 "github.com/cilium/cilium/pkg/hubble/api/v1"
+)
+
+func filterByHTTPHeader(h []*flowpb.HTTPHeader) (FilterFunc, error) {
+
+	return func(ev *v1.Event) bool {
+		for _, w := range ev.GetFlow().L7.GetHttp().GetHeaders() {
```

Comment:
sorry my mistake from a previous suggestion
```suggestion
		for _, w := range ev.GetFlow().GetL7().GetHttp().GetHeaders() {
```